<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelLgoinName">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登录名</string>
     </property>
     <property name="buddy">
      <cstring>lineSecretId</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboLine" name="lineLoginName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelSecretId">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Secret&amp;Id</string>
     </property>
     <property name="buddy">
      <cstring>lineSecretId</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSecretId">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelSecretKey">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Secret&amp;Key</string>
     </property>
     <property name="buddy">
      <cstring>lineSecretKey</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSecretKey">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelRemark">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineRemark">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkSaveSession">
     <property name="text">
      <string>记住会话</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PushButton" name="btnLogin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>src/fend/uicommon/pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ComboLine</class>
   <extends>QLineEdit</extends>
   <header>src/fend/uicommon/comboline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
